examples/connectivity/plot_mne_inverse_psi_visual.py

```diff
@@ -91,7 +91,7 @@
 tmin_con = 0.
 sfreq = epochs.info['sfreq']  # the sampling frequency
 
-psi, freqs, times, n_epochs, _ = phase_slope_index(
+psi, _, _, _, _ = phase_slope_index(
```

Comment:
no need to go back and change, but this would have been a great place for `*_`:
```suggestion
psi, *_ = phase_slope_index(
```

Reply:
This is @agramfort's preferred way of doing it as if the api changes it forces a change in your code. I would prefer actually `psi = phase_slope_index(...)[0]` in most cases if I didn't agree with the idea of making sure the API is stable